web/src/pages/balance/index.tsx

```diff
@@ -10,8 +10,68 @@ import {
   QRCodeOutlined,
   RefreshOutlined,
 } from "utils/icons";
+import VaultManager from "utils/vault-manager";
+
+interface Chain {
```

Comment:
Also I think Chain should be just the name or something , here what you really need to display is the `Coin` 

Chain is just the NativeToken on the blockchain

Basically a vault can have many coins , and then you can group the coin based on it's chain for display

Reply:
Updated using the same logic in android and ios